app/lib/dttp/params/placement_assignment.rb

```diff
@@ -92,6 +94,14 @@ def school_params
 
         params
       end
+
+      def funding_params
+        return {} unless FeatureService.enabled?("funding") && trainee.training_initiative != ROUTE_INITIATIVES_ENUMS[:no_initiative]
```

Comment:
@gpeng / @FriedSock - following on from our chat earlier, we could add a second feature flag so that we can collect but not send funding info to DTTP. Something like:

```
def funding_params
  return {} unless send_funding_to_dttp?
 ...
end

def send_funding_to_dttp?
  FeatureService.enabled?("collect_funding") && FeatureService.enabled?("send_funding_to_dttp")
end
```